nx-X11/programs/Xserver/hw/nxagent/Events.c

```diff
@@ -62,7 +62,7 @@
 #include "NXvars.h"
 #include "NXproto.h"
 
-#include "xfixesproto.h"
+#include <X11/extensions/xfixesproto.h>
 #define Window     XlibWindow
 #define Atom   XlibAtom
 #define Time XlibXID
```

Comment:
Actually no. That is, as said earlier/above, I will undo the proto stuff removal in this PR, so we need to stick with "..." header inclusion.